processing/src/test/java/org/apache/druid/segment/virtual/ExpressionVectorSelectorsTest.java

```diff
@@ -95,7 +95,7 @@
       "concat(string1, nonexistent)"
   );
 
-  private static final int ROWS_PER_SEGMENT = 100_000;
+  private static final int ROWS_PER_SEGMENT = 10_000;
```

Comment:
any reason we are adjusting this? If tests are not failing upstream we shouldn't have to make a change here.